packages/checkout/src/config.ts

```diff
@@ -1,4 +1,9 @@
 export const CHECKOUT_ENDPOINT_SBX = 'https://sbx.fung.money';
 export const CHECKOUT_ENDPOINT_PROD = 'https://app.fung.money';
 
-export const CHECKOUT_LOADED_EVENT = 'CHECKOUT_LOADED';
+export const CHECKOUT_EVENTS = [
+  'checkout:loaded',
+  'checkout:expired',
```

Comment:
same comment related to the `expired` vs `invalid`, i think invalid is better suited and more extendable